SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc

```diff
@@ -12,6 +12,147 @@ The content for this appendix may be pushed to SDPi 1.x.
+
+How does this address the "interoperability trust gap"?  By integrating SES directly into the specifications, especially integrating the ISO/IEC 11073-1070x standards, enabling "plug-and-trust" system product components, the SES implementation and operational requirements and responsibilities are greatly reduced, the "gap" is filled for all stakeholders, and the the goals of improved safety, security and clinical effectiveness of technology can be readily realized.
+
+=== SDPi Requirements Modeling & Integration
+
+As pointed out above, requirements interoperability (RI) based on robust model-based metadata is a core, innovative aspect of this SDPi profiles specification.  Given the ultimate intent to realize this description as a _Model Centric (MC) single-source-of-truth, computable, simulatable, verifiable and validatable system of systems interoperability specification_, and recognizing that it will take a significant transition period from a document-centric approach to a model-centric approach, a simplified requirements model is provided below but is aligned with the <<omg_sysml_2-0_spec>> section 7.20 Requirements language.  Of course, that specification provides for significantly more detailed and complext modeling, the general constructs may be used in this document to start the transition toward that model.  Note that SysML 2.0 also better supports model interoperability (tool-independent model exchange) and _Model-based Systems Engineering (MBSE)_ (see https://en.wikipedia.org/wiki/Model-based_systems_engineering[MBSE Wikipedia article and references]), as well as the <<ihe_eu_experience_2021_presentation_cooper_schlichting>> for an overview presentation of MBSE, MedTech system V&V, and IHE Conformity Assessment.
+
+It should be further noted that though conformity testing aspects are beyond this SDPi specification, the modeling constructs used below will also be integrated with <<omg_sysml_2-0_spec>> section 7.23 Verification Cases, to provide for advanced V&V of interoperable system components and entire systems of products.
+
+==== SDPi Requirements Core Model
+
+To formally integrate requirements in to this specification, the following requirements model provides the starting point:
+
+.SDPi Requirements - Core Model
+
+image::../images/vol1-diagram-sdpi-req-types-model.svg[align=center]
+
+This model identifies a set of requirement "types" that are formalized in the specification.  Each type is a source of requirements that are explicitly identified and formalized with appropriate metadata.
+
+[%autowidth]
+[cols="^1,4,^1,^1"]
+|===
+|Model Element |Description |AsciiDoc Attribute |Further Specified
+
+| SDPi Requirement
+| A defined stakeholde-imposed constraints that must be satisfied for a design solution to be valid.  This is an {abstract} element.
```

Comment:
What do you mean by `{abstract}`?

Reply:
Updated.  For the UML model, the top object is never instantiated.  So ... standard UML use of the term {abstract}.  In IHE, there are varying conventions for this - like the SOMDS_PARTICIPANT abstract actor in the SDPi-P actor model.  It is only realized as either a provider or a consumer.  For the work that you're doing in TF-2, this would be addressed by the "Tech Feature" metadata spec, which I'll probably include in this section. OK?

Reply:
Updated the content for clarity:  **_This is an {abstract} class model element._**